cloud/pkg/cloudhub/servers/quicserver/server.go

```diff
@@ -5,13 +5,15 @@ import (
 	"crypto/x509"
 	"fmt"
 
-	bhLog "github.com/kubeedge/beehive/pkg/common/log"
+	"k8s.io/klog"
+
 	"github.com/kubeedge/beehive/pkg/core/context"
+	"github.com/kubeedge/viaduct/pkg/api"
+	"github.com/kubeedge/viaduct/pkg/server"
+
```

Comment:
you can make all `kubeedge/` repos together, no need split them into different import sections